lib/api/user_api.dart

```diff
@@ -8,6 +8,8 @@ import 'package:client_common/api/request_models/validate_dev_request.dart';
 import 'package:client_common/api/request_models/validate_user_request.dart';
 import 'package:client_common/api/response_models/auth_response.dart';
 import 'package:client_common/api/response_models/empty_response.dart';
+import 'package:flutter/foundation.dart';
```

Comment:
Maybe add the `show` keywork for this import to limit what imported.

```suggestion
import 'package:flutter/foundation.dart' show kIsWeb;
```

Reply:
But not required